kernel/comps/virtio/src/lib.rs

```diff
@@ -9,6 +9,8 @@
 
 extern crate alloc;
 
+mod systree;
```

Comment:
I appreciate the good intention of adding this module to show that the systree and sysfs work by using Virtio as an example. But the implementation here is quite incomplete. Yes, it adds an `/sys/bus/virtio` inode. But it does not support registering any virtio device and creating the corresponding inodes below `/sys/bus/virtio/`. So I suggest leaving the virtio sysfs support to a future PR so that it can be reviewed and discussed properly.